tools/perf/ucc_pt_coll_alltoallv.cc

```diff
@@ -35,23 +35,29 @@ ucc_status_t ucc_pt_coll_alltoallv::init_coll_args(size_t count,
     ucc_status_t st      = UCC_OK;
 
     args = coll_args;
-    UCCCHECK_GOTO(ucc_mc_alloc((void**)&args.src.info_v.counts,
```

Comment:
So, this comment applies to ALL gtest test: we should use just ucc_malloc for all "counts" and "displs" allocations (see we pass UCC_MEMOrY_TYPE_HOST explicitly)